.github/pull_request_template.md

```diff
@@ -1,6 +1,3 @@
-### Ticket
-[Ticket number](link to ticket)
```

Comment:
That could be a [GH issue](https://github.com/finos/symphony-bdk-java/issues)?

Reply:
Using [those keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for instance